modules/nested-tabs-html/assets/js/frontend/handlers/nested-tabs-html.js

```diff
@@ -18,7 +18,7 @@ export default class NestedTabsHtml extends Base {
 	 * @return {string}
 	 */
 	getTabTitleFilterSelector( tabIndex ) {
-		return `[data-tab="${ tabIndex }"]`;
+		return `[data-tab-index="${ tabIndex }"]`;
```

Comment:
@hein-obox Why did we need to add the `index`?

Reply:
@louiswol94 It was a personal preference. I found the attribute name `data-tab` a bit vague.